appserver/persistence/entitybean-container/src/main/java/org/glassfish/persistence/ejb/entitybean/container/EntityContainer.java

```diff
@@ -504,55 +522,46 @@ public void appendStats(StringBuilder sbuf) {
     //Methods of EntityBeanStatsProvider
 
     public int getMaxCacheSize() {
-	int maxSize = 0;
-	if (readyStore != null) {
-	    maxSize = (cacheProp.maxCacheSize <= 0)
-		? Integer.MAX_VALUE
-		: cacheProp.maxCacheSize;
-	}
-
-	return maxSize;
+        int maxSize = 0;
+        if (readyStore != null) {
+            maxSize = (cacheProp.maxCacheSize <= 0) ? Integer.MAX_VALUE : cacheProp.maxCacheSize;
```

Comment:
why not return that directly?

Reply:
I avoided to make more changes in this class, it would take me whole day and yet another after review ...